### ISSUE TITLE
FileNotFoundError: [WinError 3] The system cannot find the path specified ... .azw3

### ISSUE THREAD

**Issue by appmu** [noDRM/DeDRM_tools#180]
### Question / bug report

Hello,

I'm not sure if this is a problem of this plugin or Calibre itself.

When I use the `calibredb` command with certain `azw` files which seems to be `azw3` internally, the following error occurs. Those files cannot be added to Calibre.

```
DeDRM v10.0.3: Trying to decrypt B09JN6TLWJ.azw
Using Library AlfCrypto DLL/DYLIB/SO
MobiDeDrm v1.1.
Copyright 2008-2020 The Dark Reverser, Apprentice Harper et al.
Decrypting Kindle Format 8 ebook: Web Development and Design for Beginners: Learn and Apply the Basic of HTML5, CSS3, JavaScript, jQuery, Bootstrap, DOM, UNIX Command and GitHub - Tools For Building Responsive Websites
Got DSN key from database default_key
Found 4 keys to try after 0.2 seconds
Crypto Type is: 0
This book is not encrypted.
Decryption succeeded after 0.2 seconds
DeDRM v10.0.3: Finished after 0.2 seconds
Traceback (most recent call last):
  File "calibre\ebooks\metadata\worker.py", line 67, in run_import_plugins
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\...\\AppData\\Local\\Temp\\calibre_x_44js5t\\ljudxoh0.azw3' -> 'C:\\Users\\...\\AppData\\Local\\Temp\\calibre_x_44js5t\\08xz8sz0add-single\\1\\B09JN6TLWJ.azw3'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "runpy.py", line 196, in _run_module_as_main
  File "runpy.py", line 86, in _run_code
  File "site.py", line 82, in <module>
  File "site.py", line 77, in main
  File "site.py", line 49, in run_entry_point
  File "calibre\db\cli\main.py", line 252, in main
  File "calibre\db\cli\main.py", line 40, in run_cmd
  File "calibre\db\cli\cmd_add.py", line 474, in main
  File "calibre\db\cli\cmd_add.py", line 239, in do_add
  File "calibre\db\cli\main.py", line 184, in run
  File "calibre\db\cli\cmd_add.py", line 180, in implementation
  File "calibre\db\cli\cmd_add.py", line 121, in book
  File "calibre\db\adding.py", line 125, in run_import_plugins
  File "calibre\ebooks\metadata\worker.py", line 69, in run_import_plugins
  File "shutil.py", line 256, in copyfile
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\...\\AppData\\Local\\Temp\\calibre_x_44js5t\\08xz8sz0add-single\\1\\B09JN6TLWJ.azw3'
```

If I manually drag-and-drop the file to the Calibre UI, it is added with a blank cover with a warning.

```
calibre, version 6.6.1
WARNING: Errors while adding: There were problems adding some files, click "Show details" for more information


----------------------------------------------------------------------
Failed to read metadata from the file:
	\\?\...\B09JN6TLWJ.azw
With error:
Traceback (most recent call last):
  File "calibre\ebooks\metadata\worker.py", line 67, in run_import_plugins
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Users\\...\\AppData\\Local\\Temp\\calibre_165phw7w\\2fxd7wt5.azw3' -> 'C:\\Users\\...\\AppData\\Local\\Temp\\calibre_165phw7w\\on5r05q8_add_books\\0\\B09JN6TLWJ.azw3'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "calibre\utils\ipc\pool.py", line 371, in worker_main
  File "calibre\ebooks\metadata\worker.py", line 79, in read_metadata
  File "calibre\ebooks\metadata\worker.py", line 69, in run_import_plugins
  File "shutil.py", line 256, in copyfile
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\...\\AppData\\Local\\Temp\\calibre_165phw7w\\on5r05q8_add_books\\0\\B09JN6TLWJ.azw3'
```

### Which version of Calibre are you running?

6.6.1

### Which version of the DeDRM plugin are you running?

v10.0.3

### If applicable, which version of the Kindle software are you running?

Kindle of PC 1.30.0

### Log output

```log
Paste log output here.
```

**Comment by ElleKayEm** [comment]
Does not appear to be a DeDRM problem.  The book isn't encrypted anyway.  You could see if disabling the plugin changes anything.

**Comment by appmu** [comment]
Strangely, it's working now.

1. Deactivated the plugin and dropped the file to Calibre UI -> the book was normally added with a cover image. I deleted the book from Calibre.
2. Tested whether the `calibredb` command works without the plugin and it added the book without an error. I deleted the book from Calibre. 
3. Checked to see if the `calibredb` command works with this plugin activated. Strangely, it worked this time. 

I'm not sure what was going on.
